### PR TITLE
Improves documentation

### PR DESCRIPTION
**Pull request by bencorlett**
By improving our documentation, we'll reduce friction in the onboarding experience and also assist with internal testing.

- [x] Rewrite custom-facing documentation.
- [ ] Rewrite development documentation/guidelines.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,164 +1,336 @@
...
+The module's settings are controlled within `Stores -> Configuration -> Services -> AddressFinder`.
+
+> Most settings in Magento 2 are guarded with sensible defaults. To customise settings,  you'll need to uncheck the **use system value** for any settings you would like to customise.
+
+### 3.1 Basic functionality
+
+ To get the module functioning in its most basic form, you'll need to:
 
+1. Change **Enabled** to **Yes**.
+2. Obtain a licence key for [Australia](https://portal.addressfinder.io/signup/au?utm_source=magento&utm_medium=extension&utm_campaign=magento_admin&utm_term=Australia) or [New Zealand](https://portal.addressfinder.io/signup/nz?utm_source=magento&utm_medium=extension&utm_campaign=magento_admin&utm_term=New Zealand) and enter this in the **License** field.
```
Comment (bencorlett) [inline README.md:107]: @katenorquay could you please clarify the GA `utm_*` fields?

**Review by katenorquay — APPROVED** [review]
Looks awesome Ben!

File: README.md (hunk 1 above)
Comment (1mattramsay) [reply README.md:107]: These look correct. They tell our google analytics that a customer came to our website after clicking one of these links.

Comment (bencorlett) [reply README.md:107]: @1mattramsay yeo, but the GA codes were copied from system config in Magento - do you want to change these to reflect coming from GitHub?

**Pull request merged**